### ISSUE TITLE
the number of UAVs

### ISSUE THREAD

**Issue by W1416167256** [dmslab-konkuk/LogisticsEnv#11]
Hello, I am very interested in this project, may I ask how to set the number of UAVs to create a bigger scene.

**Comment by leehe228** [comment]
Hi, thank you for your interest in my research.

Currently, you need to change the number of Agents and the constant(`5`) in some code in our Unity Project yourself.

We are working on a new version with support for arguments and config files, which we hope to share with you as soon as possible.

Thank you.

**Comment by W1416167256** [comment]
Do I need to recompile after modifying these constant?

**Comment by leehe228** [comment]
Yes, the provided build file is that with the number of drones fixed at 5. Any modifications to the Unity project and code should be built within Unity for the specific operating system. 

The config settings in main.py will apply regardless of the build file.
